src/palette/qml/MuseScore/Palette/internal/Palette.qml

```diff
@@ -526,7 +526,7 @@ StyledGridView {
             }
 
             onClicked: {
-                if (!paletteView.paletteController.applyPaletteElement(paletteCell.modelIndex, ui.keyboardModifiers())) {
+                if (paletteView.paletteController.applyPaletteElement(paletteCell.modelIndex, ui.keyboardModifiers())) {
```

Comment:
Here is where I noticed the active index being lost after, for example, mouse selection of a palette item
Without removing the negate operator, the index would revert back to be the first thing on the palettes. Not sure how/why this code had the negate to begin with. Probably a copy error